### PR TITLE
Return 2xx on already exists association

### PR DESCRIPTION
**Pull request by Zorin95670**

### CODE REVIEW

File: src/test/java/com/ditrit/letomodelizerapi/service/AccessControlServiceImplTest.java [hunk 1]
```diff
@@ -240,8 +241,8 @@ void testAssociateUser() {
     }
 
     @Test
-    @DisplayName("Test associateUser: should throw exception on already existing association")
-    void testAssociateUserThrow() {
+    @DisplayName("Test associateUser: should do Nothing on already existing association")
```
Comment (justinehell) [inline src/test/java/com/ditrit/letomodelizerapi/service/AccessControlServiceImplTest.java:244]: do nothing

File: src/test/resources/features/Group.feature [hunk 2]
```diff
@@ -170,17 +170,11 @@ Feature: group feature
       | [group_test_3_id] | text |
     Then I expect "201" as status code
 
-    # Should not be able to create cycle in group hierarchy
+    # Should not be able to create cycle in group hierarchy but return 200
```
Comment (justinehell) [inline src/test/resources/features/Group.feature:173]: 200 or 201? (line 177 I expect "201" as status code)

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/test/java/com/ditrit/letomodelizerapi/service/AccessControlServiceImplTest.java (hunk 1 above)
Comment (Zorin95670) [reply src/test/java/com/ditrit/letomodelizerapi/service/AccessControlServiceImplTest.java]: done

File: src/test/resources/features/Group.feature (hunk 2 above)
Comment (Zorin95670) [reply src/test/resources/features/Group.feature]: done

**Review by justinehell — APPROVED** [review]

**Pull request merged**